test/spec/flink_job.rb

```diff
@@ -6,7 +6,6 @@
 
 class FlinkJob
 
-  TRAVIS_BRANCH = ENV['TRAVIS_BRANCH']
```

Comment:
Does this need to be replaced?

Reply:
It actually wasn't being used anywhere in the class.